src/Plivo/Resource/Message/MessageInterface.cs

```diff
@@ -176,6 +176,161 @@ public MessageInterface(HttpClient client) : base(client)
             return result.Object;
         }
 
+        #endregion
+        
+        //Handling dst(string)
+        #region Create
+        /// <summary>
+        /// Create Message with the specified src, dst, text, type, url, method , log, media_urls, media_ids.
+        /// </summary>
+        /// <returns>The create.</returns>
+        /// <param name="src">Source.</param>
+        /// <param name="dst">Dst.</param>
+        /// <param name="text">Text.</param>
+        /// <param name="type">Type.</param>
+        /// <param name="url">URL.</param>
+        /// <param name="method">Method.</param>
+        /// <param name="log">Log.</param>
+        /// <param name="trackable">trackable.</param>
+        /// <param name="powerpackUUID">powerpackUUID</param>
+        ///<param name="media_urls">media_urls</param>
+        ///<param name="media_ids">media_ids</param>
+        public MessageCreateResponse Create(
```

Comment:
can we handle both string or List in same function? Lot of repeated duplicate code here